### PR TITLE
arduino: add examples

### PR DESCRIPTION
**Pull request by Ratheshan03**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by kbdharun** [comment]
@Ratheshan03 Thanks for your contribution. We already have this page in [common](https://github.com/tldr-pages/tldr/blob/main/pages/common/arduino.md) feel free to edit it.

**Comment by Ratheshan03** [comment]
@kbdharun,  updated arduino command in common folder with new examples.

File: pages/common/arduino.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # arduino
 
 > Arduino Studio - Integrated Development Environment for the Arduino platform.
-> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>.
+> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>, <https://manned.org/arduino>.
```
Comment (kbdharun) [inline pages/common/arduino.md:4]:
```suggestion
> More information: <https://github.com/arduino/Arduino/blob/master/build/shared/manpage.adoc>.
```
We use manned as a fallback when official documentation doesn't exist. But here link is already present and our clients support only 1 link in this line, so I guess we can remove it.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

Is it possible to join 2 examples as our pages only supports a maximum of 8 examples, if you think it is necessary feel free to create a seperate page for that particular flag similar to `brew-cask`.

File: pages/common/arduino.md (hunk 1 above)
Comment (Ratheshan03) [reply pages/common/arduino.md]: updated as you suggested 👍🏼

File: pages/common/arduino.md [hunk 2]
```diff
@@ -26,3 +26,11 @@
 - Save any (changed) preferences to `preferences.txt`:
 
 `arduino --save-prefs`
+
+- Install latest SAM board support:
+
+`arduino --install-boards "arduino:sam"`
```
Comment (EmilySeville7cfg) [inline pages/common/arduino.md:32]: `{{...}}` can be used to underline that `--install-boards` accepts any valid value and not just `arduino:sam`:

```suggestion
`arduino --install-boards "{{arduino:sam}}"`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/arduino.md (hunk 2 above)
Comment (Ratheshan03) [reply pages/common/arduino.md]: @EmilySeville7cfg, requested changes have been updated!

File: pages/common/arduino.md [hunk 3]
```diff
@@ -26,3 +26,11 @@
 - Save any (changed) preferences to `preferences.txt`:
 
 `arduino --save-prefs`
+
+- Install latest SAM board support:
```
[the remark this answers is not in the document]
Comment (Ratheshan03) [reply pages/common/arduino.md]: support is not necessary 👍🏼

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**